packages/ui/src/components/eventDetails.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @author Thenmozhi Subramaniam
+ * @name EventDetails
+ * @desc renders event details component
+ */
+
+import React, {Component} from 'react';
+
+class EventDetails extends Component
+{
+  componentDidMount =  () => {
+     this.props.getEventDetails(this.props.match.params.id,this.props.userInfo.id);
+  }
+  constructor(props){
+    super(props);
+  }
+
+  renderBreadCrumb = () => {
+    console.log(this.props);
+    return this.state.BreadCrumb.map((item, index) => {
```

Comment:
Does this even work? Where is the state variable? you are supposed to declare and initialize them in constructor